eva_cttv_pipeline/evidence_string_generation/consequence_type.py

```diff
@@ -63,100 +66,12 @@ class SoTerm(object):
     Holds information on accession and rank.
     """
 
-    so_accession_name_dict = {'transcript_ablation': 1893,
```

Comment:
There are actually more terms in this dictionary than from the union of VEP consequences + repeat expansion consequences, I don't see how we could ever generate them with the current pipeline though so I've removed them...

Reply:
Created #324 to implement a better way of handling this.

Reply:
Good idea 👍 

Out of curiosity: before the next release we need to address this issue, otherwise the pipeline will be missing terms, right?

Reply:
I don't think so, because the pipeline can't generate any consequences besides what VEP returns, or the two repeat expansion terms that I've [kept in](https://github.com/EBIvariation/eva-opentargets/blob/8d4079266854ec9e09de8d24a9b7d9d5a5abc0a7/eva_cttv_pipeline/evidence_string_generation/consequence_type.py#L68).

I'd probably try to do it before the next release anyway, because it goes along with this and other consequence severity ranking things we've been thinking about...

Reply:
Great then. I was wondering if we only had two extra repeat expansion consequences, but I have no further doubts